src/tests.rs

```diff
@@ -0,0 +1,19 @@
+use crate::COMLibrary;
+use crate::WMIConnection;
+
+pub mod fixtures {
+    use super::*;
+    use lazy_static::lazy_static;
+    use std::rc::Rc;
+
+    // This way we only setup COM security once during tests.
+    lazy_static! {
+        static ref COM_LIB: COMLibrary = { COMLibrary::new().unwrap() };
```

Comment:
you could probably get away with `https://doc.rust-lang.org/std/sync/struct.Once.html`